pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java

```diff
@@ -371,6 +373,19 @@ public void healthcheck(@Suspended AsyncResponse asyncResponse) throws Exception
                     asyncResponse.resume("ok");
                 }
             });
+
+    }
+
+    @GET
+    @Path("/health")
```

Comment:
shouldn't this move to v1-brokers.?

Reply:
that was there originally don't want to change the default behavior.

Reply:
or we can use just a query-param with topic format, if it doesn't exist then default behavior , if it does then it formats topic based on the format. that might be simple and clean change?